test/init_helm_promise_test.go

```diff
@@ -110,18 +118,67 @@ var _ = Describe("init helm-promise", func() {
 				matchPromise(workingDir, "postgresql", "syntasso.io", "v1alpha1", "Database", "database", "databases")
 			})
 
+			By("including CRD schema from chart values in promise.yaml", func() {
+				props := getCRDProperties(workingDir, false)
+				matchExampleHelloWorldHelmChartSchema(props)
+			})
+
 			By("including helm resource configure workflow in promise.yaml", func() {
 				pipelines := getWorkflows(workingDir)["resource"]["configure"]
 				Expect(pipelines).To(HaveLen(1))
 				matchHelmResourceConfigurePipeline(pipelines[0], []corev1.EnvVar{
-					{Name: "CHART_URL", Value: "postgres.io/chart"},
-					{Name: "CHART_NAME", Value: "greatchart"},
-					{Name: "CHART_VERSION", Value: "v110.0.1"},
+					{Name: "CHART_URL", Value: "https://helm.github.io/examples"},
+					{Name: "CHART_NAME", Value: "hello-world"},
+					{Name: "CHART_VERSION", Value: "0.1.0"},
 				})
 			})
 		})
 	})
 
+	Context("helm chart integration", func() {
+		It("works with OCI helm chart", func() {
```

Comment:
since OCI takes a few seconds, worth adding some log output? "Pulling down OCI image..." or "Fetching helm chart..."

Reply:
discussed and not going to add more log output